tests/done_callback/package.json

```diff
@@ -3,8 +3,8 @@
 	"version": "1.0.0",
 	"description": "An example showing how Jazzer.js handles callback based fuzz targets",
 	"scripts": {
-		"fuzz": "jazzer fuzz",
-		"dryRun": "jazzer fuzz -- -runs=1 -seed=123456789"
+		"fuzz": "jazzer fuzz -x Error -- -max_total_time=60",
```

Comment:
Any reason we use a `max_total_time=60` here compared to a `max_total_time=120` in the `FuzzedDataProvider`?

Reply:
On my system, most tests are quite fast, so `60` should be a save value. Not too long in case of a problem, but long enough for the slower CI systems to finish. On the other hand the `FuzzedDataProvider` test is pretty slow and took longer than `60` seconds on my machine, hence, the longer timeout for CI.